deploy/k8s-cheat-sheet.md

```diff
@@ -0,0 +1,310 @@
+### Minikube information
+
+Check local environment minikube version:
+
+```bash
+> minikube version
+minikube version: v0.17.1
+```
+
+Start minikube:
+
+```bash
+> minikube start
+Starting local Kubernetes cluster...
+Starting VM...
+SSH-ing files into VM...
+Setting up certs...
+Starting cluster components...
+Connecting to cluster...
+Setting up kubeconfig...
+Kubectl is now configured to use the cluster.
+```
+
+### Cluster info
+
+See both client and server k8s version:
+
+```bash
+> kubectl version --short
+Client Version: v1.5.3
+Server Version: v1.5.3
+```
+
+Get information about endpoints for k8s master, DNS and dashboards
+```bash
+> kubectl cluster-info
+Kubernetes master is running at https://192.168.99.100:8443
+KubeDNS is running at https://192.168.99.100:8443/api/v1/proxy/namespaces/kube-system/services/kube-dns
+kubernetes-dashboard is running at https://192.168.99.100:8443/api/v1/proxy/namespaces/kube-system/services/kubernetes-dashboard
+```
+
+View list of nodes that are part of the k8s cluster:
+
+```bash
+> kubectl get nodes
+NAME       STATUS    AGE
+minikube   Ready     76d
+```
+
+### Contexts
+
+Get list of configured contexts:
+
+```bash
+> kubectl config get-contexts
+CURRENT   NAME                CLUSTER           AUTHINFO                NAMESPACE
+*         minikube            minikube          minikube
+```
+
+Update the current context with a namespace:
+
+```bash
+> kubectl config set-context $(kubectl config view | awk '/current-context/ {print $2}') --namespace=call-roulette
+Context "minikube" set.
+
+> kubectl config get-contexts
+CURRENT   NAME                CLUSTER           AUTHINFO                NAMESPACE
+*         minikube            minikube          minikube                call-roulette
+
+```
+
+### Deployment
+
+Deploy an app:
+
+```bash
+> kubectl run <deployment-name> --image=<docker-image>:<image-version> --port=<port-to-be-used-by-the-app>
```

Comment:
yes, it's in the README -> https://github.com/saghul/CallRoulette/pull/7/files#diff-bcdcc71d7dab57472393a413b9f8293dR32